x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/table/no_entities_empty_state.tsx

```diff
@@ -50,6 +52,34 @@ export function NoEntitiesEmptyState() {
 
   return (
     <EuiFlexGroup direction="column">
+      {!userHasDismissedCallout && (
+        <EuiFlexItem>
+          <EuiCallOut
+            css={{ textAlign: 'left' }}
+            onDismiss={() => setUserHasDismissedCallout(true)}
+            title={i18n.translate('xpack.apm.noEntitiesEmptyState.callout.title', {
+              defaultMessage: 'Trying for the first time?',
+            })}
+          >
+            <p>
+              {i18n.translate('xpack.apm.noEntitiesEmptyState.description', {
+                defaultMessage:
+                  'It can take up to a couple of minutes for your services to show. Try refreshing the page in a minute. ',
```

Comment:
We can remove the extra space here:

```suggestion
                  'It can take up to a couple of minutes for your services to show. Try refreshing the page in a minute.',
```